src/presentational/date-nav/DateNavigator.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react' 
+import './DateNavigator.css';
+import Util from '../../Util';
+export default class DateNavigator extends Component {
```

Comment:
this allows the user to move a single day ahead or back to brows entries in that day.
it receives the call backs for when user hits forward or backward.